packages/ppx/src/css_to_emotion.re

```diff
@@ -374,7 +374,7 @@ and render_selectors = selectors => {
   |> List.map(((selector, _loc)) => render_selector(selector))
   |> String.concat(", ");
 }
-and render_style_rule = (ident, rule: style_rule) => {
+and render_style_rule_parts = (rule: style_rule) => {
```

Comment:
How about if this is defined in `render_style_rule` function and perhaps re-named?

Reply:
I also was thinking to define it inside `render_style_rule` but it is also used on `render_style_rules` 🤔
hehe need to find a better name xD

Reply:
hehe yeah, need to find a better name xD

Reply:
> I also was thinking to define it inside render_style_rule but it is also used on render_style_rules 🤔

Right, I didn't notice that.

Reply:
yup definitely better